force-app/main/default/classes/AsynchronousTest.cls

```diff
@@ -30,7 +30,7 @@ public with sharing class AsynchronousTest {
     static void nextStepFuture_testUpdateAllNextStep() {
         // Create test data
         List<Account> testAccounts = new List<Account>();
-        for (Integer i = 0; i < 5; i++) {
+        for (Integer i = 0; i < 500; i++) {
```

Comment:
- Not a problem but be careful with how many records you use to test your solutions to avoid test errors with batches or queuables - See #4 here - https://apexstepbystep.com/queueable-apex/